Makefile

```diff
@@ -125,6 +126,10 @@ test_SPM:
 	$(SWIFT) package clean
 	$(SWIFT) test
 
+test_docker:
+	$(DOCKER) build --tag grdb .
```

Comment:
Please guard $(DOCKER) with `ifdef`, because if Docker is not installed then $(DOCKER) is empty and the Makefile executes weird commands. See example with $(POD) in the Makefile.

Maybe there is a better way to write Makefiles - but until we figure it out, let's use this convention for tools that may be missing.